R/FunctionReporter.R

```diff
@@ -304,6 +304,7 @@ FunctionReporter <- R6::R6Class(
                             , pkg_env = private$get_pkg_env()
                             , pkg_functions = funs
                             )
+                        , SIMPLIFY = FALSE
```

Comment:
R's aggressive default coercing strikes again! :shakefist:

Great job catching this problem!